client/engine/chainservice/chainservice.go

```diff
@@ -6,14 +6,57 @@ import (
 	"github.com/statechannels/go-nitro/types"
 )
 
-// ChainEvent is an internal representation of a blockchain event
-type Event struct {
+// Event dictates which methods all chain events must implement
+type Event interface {
+	GetChannelId() types.Destination
```

Comment:
nit, but a nit from [on high](https://go.dev/doc/effective_go) (ctrl+f getters):

> Go doesn't provide automatic support for getters and setters. There's nothing wrong with providing getters and setters yourself, and it's often appropriate to do so, but it's neither idiomatic nor necessary to put Get into the getter's name. If you have a field called owner (lower case, unexported), the getter method should be called Owner (upper case, exported), not GetOwner

```suggestion
	ChannelID() types.Destination
	BlockNum() uint64
```

Reply:
(I've lost the documentation, but `ID` is considered an initialism by picky Go persons, and should be denoted either as `id` or `ID`)

Reply:
Makes sense to me. Addressed in 1e939c8.

Reply:
Also, `GetBlockNum` has been removed in 1ee011a.